data/ships/bowfin.json

```diff
@@ -7,39 +7,107 @@
 	"ship_class": "light_fighter",
 	"min_crew": 1,
 	"max_crew": 1,
-	"price": 34430,
-	"hull_mass": 25,
+	"price": 43104,
+	"hull_mass": 20,
+	"structure_mass": 7.1,
+	"armor_mass": 11.6,
+	"volume": 135,
 	"atmospheric_pressure_limit": 6.5,
-	"capacity": 18,
-	"slots": {
-		"engine": 1,
-		"cabin": 0,
-		"scoop": 0,
-		"laser_front": 1,
-		"missile": 24,
-		"cargo": 6
+	"capacity": 15,
+	"cargo": 4,
+
+	"equipment_slots": {
+		"missile_bay_1": {
+			"type": "missile_bay.bowfin_internal",
+			"size": 2,
+			"i18n_key": "HARDPOINT_MISSILE_ARRAY",
+			"tag": "tag_missile_bay_1",
+			"hardpoint": true,
+			"default": "missile_rack.bowfin_internal_s2"
```

Comment:
What is this "default" for?  Why is is "missile_rack.bowfin_xxx" and not "missile_bay.bowfin_xxx"?

Reply:
Didn't get updated when the ID of the equipment was changed. `default` indicates equipment that "should be installed by default", though that's still nebulous as nothing currently uses the key. It's planned for implementation in Stage 2.

Reply:
Amended on Notion, will re-import post-merge with other fixes.